Readme.md

```diff
@@ -1 +1,78 @@
-# Discover
+# [Liquid Investigations](https://github.com/liquidinvestigations): Node
+Discovery Daemon
+
+This repository implements a method finding out in real time what other Liquid
+Investigations nodes are online.
+
+Ths discovery service must be run as `root`.
+
+
+## The basic principle
+
+[Zeroconf](https://en.wikipedia.org/wiki/Zero-configuration_networking) is
+being used for discovery of the other nodes in a network. Each node publishes
+itself as a `_liquid._tcp.local` service using
+[avahi-daemon](https://packages.debian.org/jessie/avahi-daemon).
+
+Each node in the network must have a unique hostname by which it will be
+identified. For the test, the hostnames used are `vagrant-box-two.liquid`,
+`vagrant-box-two.liquid`, and `vagrant-box-three.liquid`. The hostnames can
+have any domain, like `my-pretty-box.me` or `the-cool-box.org`. The hostnames
+should consist only of smallcase letters and the hyphen ('-'). The hostnames
```

Comment:
Marked this in [this issue](https://github.com/liquidinvestigations/discover/issues/4)